### PR TITLE
Incorporated O'Reilly changes to /trusted-committer/06-advocating-for-the-communitys-needs

### PR DESCRIPTION
**Pull request by Ludmila-N**
Incorporated O'Reilly changes to /trusted-committer/06-advocating-for-the-communitys-needs.asciidoc

### CODE REVIEW

File: trusted-committer/06-advocating-for-the-communitys-needs.asciidoc [hunk 1]
```diff
@@ -1,49 +1,31 @@
...
-medium term results of the community.  InnerSource communities, on the other
-hand understand that a healthy community is a precondition for healthy code and
-are naturally more concerned with the longevity of both the product and the
-community. This is why many InnerSource initiatives were modeled after the
-Apache Way, which has the motto "Community over Code".
+InnerSource communities exist in a corporate context and are thus more constrained than Open Source communities. Sometimes the
+business unit’s interests are at odds with those of the community.
+Trusted Committers take a long term perspective on their project. 
+They understand that a healthy community is a precondition for healthy code. 
+This is why many InnerSource initiatives were modeled in the Apache Way with it's http://theapacheway.com/community-over-code/["Community over Code"] motto.
```
Comment (rrrutledge) [inline trusted-committer/06-advocating-for-the-communitys-needs.asciidoc:9]:
```suggestion
This is why many InnerSource initiatives were modeled in the Apache Way with its http://theapacheway.com/community-over-code/["Community over Code"] motto. 
```

File: trusted-committer/06-advocating-for-the-communitys-needs.asciidoc [hunk 2]
```diff
@@ -1,49 +1,31 @@
...
-are naturally more concerned with the longevity of both the product and the
-community. This is why many InnerSource initiatives were modeled after the
-Apache Way, which has the motto "Community over Code".
+InnerSource communities exist in a corporate context and are thus more constrained than Open Source communities. Sometimes the
+business unit’s interests are at odds with those of the community.
+Trusted Committers take a long term perspective on their project. 
+They understand that a healthy community is a precondition for healthy code. 
+This is why many InnerSource initiatives were modeled in the Apache Way with it's http://theapacheway.com/community-over-code/["Community over Code"] motto. 
+Business units, on the other hand, are naturally more concerned with the products produced by an InnerSource community. 
+They prefer to see short- to medium-term results that help the bottom line.
```
Comment (rrrutledge) [inline trusted-committer/06-advocating-for-the-communitys-needs.asciidoc:11]:
```suggestion
They prefer to see short to medium-term results that help the bottom line.
```

File: trusted-committer/06-advocating-for-the-communitys-needs.asciidoc [hunk 3]
```diff
@@ -1,49 +1,31 @@
...
-public credit in this case. Failure to give credit is almost never done in bad
-faith, though, and Trusted Committers should be able to easily correct that.
+Trusted Committers also need to make sure that the community and individual https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_contributors_] get public credit for their work. 
+Public credit is the currency with which contributors are being paid, especially those who contribute voluntarily. 
+It is a good practice to commend valuable contributors publicly and to make sure their managers are aware of their contributions.
+Neglecting to give credit can be frustrating for individual contributors and detrimental to the health of the community. 
+This can happen in companies not yet accustomed to the InnerSource working model, or when the software being developed by the InnerSource community runs _behind the scenes_ and managers were simply not aware of the community’s contribution. 
+A good Trusted Committer will engage with management and advocate for public credit. 
+Failure to give credit is almost never done in bad faith and is usually easy to fix.
```
Comment (rrrutledge) [inline trusted-committer/06-advocating-for-the-communitys-needs.asciidoc:24]:
```suggestion
Failure to give credit is almost never done in bad faith and is easy to fix.
```

File: trusted-committer/06-advocating-for-the-communitys-needs.asciidoc [hunk 4]
```diff
@@ -1,49 +1,31 @@
...
-managers are CC'd as well. Neglecting to give credit, on the other hand, can be
-hugely frustrating for individual contributors and very detrimental for the
-health of the community overall. Neglecting to give credit can occur in
-companies which are not yet accustomed to the InnerSource working model or when
-the software being developed by the InnerSource community runs _behind the
-scenes_ and managers were simply not aware of the community's contribution. A
-good Trusted Committer will engage with management and advocate the need for
-public credit in this case. Failure to give credit is almost never done in bad
-faith, though, and Trusted Committers should be able to easily correct that.
+Trusted Committers also need to make sure that the community and individual https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_contributors_] get public credit for their work.
```
Comment (lenucksi) [inline trusted-committer/06-advocating-for-the-communitys-needs.asciidoc:18]: @tsadler1988 Do you have an idea how inter-article linking would work with the straight-from-GitHub rendering you made? 
Maybe use some identifier from our mapping JSON here (e.g. as non-existing-anchor in the link here) and fetch the relevant rendered URL? (I suck at JS and hence can only come up with horrible, totally not state-of-the-art hacks like that :wink: )

Comment (tsadler1988) [reply trusted-committer/06-advocating-for-the-communitys-needs.asciidoc:18]: As things stand, that link will just go to GitHub. I believe this was raised and we accepted this for the first pass.

An identifier would be ideal, but doing a string replace of each literal GitHub link to the equivalent innersourcecommons.org link should be feasible. I will try and have something to show on this by Thursday :)

Comment (lenucksi) [reply trusted-committer/06-advocating-for-the-communitys-needs.asciidoc:18]: > As things stand, that link will just go to GitHub. I believe this was raised and we accepted this for the first pass.

Sorry, I forgot that. My bad. 

> 
> An identifier would be ideal, but doing a string replace of each literal GitHub link to the equivalent innersourcecommons.org link should be feasible. I will try and have something to show on this by Thursday :)

W00t w00t! :tada:

Comment (rrrutledge) [reply trusted-committer/06-advocating-for-the-communitys-needs.asciidoc:18]: Do we have multiple final locations where we'd like these links to point?  Or will they just point to the ISC website each time?

Comment (tsadler1988) [reply trusted-committer/06-advocating-for-the-communitys-needs.asciidoc:18]: @rrrutledge are you suggesting that the `.asciidoc` files' links should always point to `innersourcecommons.org`? That seems reasonable, but could this mean less reason to keep the `urls.yaml` up to date? How would that be enforced?

Comment (rrrutledge) [reply trusted-committer/06-advocating-for-the-communitys-needs.asciidoc:18]: Yes, I am suggesting that.

Delete `urls.yaml`.

Discussion in this week's call?

Comment (lenucksi) [reply trusted-committer/06-advocating-for-the-communitys-needs.asciidoc:18]: I'm just casually throwing in here #244 as a possible second publication target.

**Comment by lenucksi** [comment]
This is a part of #157.

File: trusted-committer/06-advocating-for-the-communitys-needs.asciidoc (hunk 4 above)
Comment (Ludmila-N) [reply trusted-committer/06-advocating-for-the-communitys-needs.asciidoc:18]: ok to merge according to [last meeting notes](https://docs.google.com/document/d/1ktfIb4-5u9AmAC1dB9LhbjquuopijY1DVcN5JdFpPPs/edit)

**Comment by lenucksi** [comment]
@rrrutledge anything preventing us from merging here, now that the italics have been removed?
Wouldn't want to merge with "changes requested" votes standing.

**Pull request merged**